modules/drivers/snowflake/src/metabase/driver/snowflake.clj

```diff
@@ -353,3 +353,32 @@
 (defmethod sql-jdbc.execute/set-parameter [:snowflake java.time.ZonedDateTime]
   [driver ps i t]
   (sql-jdbc.execute/set-parameter driver ps i (t/sql-timestamp (t/with-zone-same-instant t (t/zone-id "UTC")))))
+
+;;; +----------------------------------------------------------------------------------------------------------------+
+;;; |                                         metabase.driver.sql-jdbc.execute impls                                 |
+;;; +----------------------------------------------------------------------------------------------------------------+
```

Comment:
A little weird to put a header for a 

> metabase.driver.sql-jdbc.execute impls

section when I can see a `sql-jdbc.execute` impl directly **above** this